### ISSUE TITLE
uBlock Origin Logger remembers closed private tabs 

### ISSUE THREAD

**Issue by matter-it-does** [uBlockOrigin/uBlock-issues#2473]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

![Screenshot 2023-01-30 at 20 21 34](https://user-images.githubusercontent.com/76812421/215511967-7ebe9f70-8e95-453e-a152-39aae16e517d.png)

The masked tabs are from a private window and ublock logger is in normal mode.

### A specific URL where the issue occurs.

N/A

### Steps to Reproduce

- Open private mode, 
- duck.com
- close private mode
- in normal mode, open ublock logger
- expand the tab list
- see google there

### Expected behavior

forget private mode  activity

### Actual behavior

remember private mode activity

### uBO version

1.16.0

### Browser name and version

Firefox 109

### Operating System and version

macOS

**Comment by gwarser** [comment]
https://github.com/uBlockOrigin/uBlock-issues/issues/755

**Comment by gorhill** [comment]
It's not just private tabs, I can reproduce with normal tabs too.

**Comment by gwarser** [comment]
Not only private tabs. For some reason, GitHub, Gmail and Twitter are not cleared in normal mode too.

**Comment by garry-ut99** [comment]
Also: a similiar issue in the past https://github.com/uBlockOrigin/uBlock-issues/issues/2473 has not been considered an issue.

<hr>

Getting back to the current issue - I've found something, steps to reproduce (in normal mode 1.46.1b13 + 109):
- `google.com`
- open uBO Logger
- `futbin.com`
- switch to uBO logger and  open any image request in a new tab 
(by clicking an arrow at the end of the request)
- switch to uBO logger and select `google.com`
- switch to the image tab and close it 
- switch to uBO logger and see that the image tab entry is still listed as Active tab, despite it has been closed
- in uBO logger click on the image tab entry and it will dissappear from the uBO logger

I'm not sure whether it's the same issue as [755](https://github.com/uBlockOrigin/uBlock-issues/issues/755) + [2473](https://github.com/uBlockOrigin/uBlock-issues/issues/2473) just with different steps to reproduce or it's just a similiar issue but not same, I'm even not sure whether it's considered an issue or just a feature.

**Comment by garry-ut99** [comment]
Yes, it's a duplicate, just that nobody realised in 755 that private tabs were affected too.
